### PR TITLE
Fix Nightly

### PR DESCRIPTION
**Pull request by armstrongnate**
We needed to update RN dependencies because of:
facebook/react-native#29633 (comment)

[run-nightly]

refs: MBL-14777
affects: none
release note: none

test plan: Nightly should pass

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -299,20 +299,20 @@ PODS:
...
   - react-native-document-picker (3.5.4):
-    - React
-  - react-native-image-picker (2.3.3):
-    - React
+    - React-Core
+  - react-native-image-picker (2.3.4):
+    - React-Core
   - react-native-segmented-control (2.1.1):
-    - React
+    - React-Core
```
Comment (armstrongnate) [inline Podfile.lock:315]: Yeah these are what was needed to fix the build.

File: rn/Teacher/package.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
     "react-native-audio": "^3.5.1",
     "react-native-button": "^2.1.0",
     "react-native-camera": "^3.23.1",
-    "react-native-document-picker": "Elyx0/react-native-document-picker",
+    "react-native-document-picker": "instructure/react-native-document-picker",
```
Comment (armstrongnate) [inline rn/Teacher/package.json:55]: Having to fork rn dependencies for things like this is mega sad.

File: rn/Teacher/package.json [hunk 3]
```diff
@@ -28,11 +28,11 @@
     "fsevents": "^1.2.4"
   },
   "dependencies": {
-    "@react-native-community/art": "^1.2.0",
+    "@react-native-community/art": "instructure/art",
     "@react-native-community/async-storage": "^1.6.1",
     "@react-native-community/datetimepicker": "^3.0.0",
     "@react-native-community/picker": "^1.6.6",
-    "@react-native-community/segmented-control": "^2.1.1",
+    "@react-native-community/segmented-control": "instructure/segmented-control",
```
Comment (vanwagonet) [inline rn/Teacher/package.json:35]: `react-native-segmented-control/segmented-control` has the fix, I don't think we need our own fork of this.

**Review by vanwagonet — CHANGES_REQUESTED** [review]
Let avoid a fork where we can, & submit a PR to the upstreams where we have forked.

File: rn/Teacher/package.json (hunk 3 above)
Comment (armstrongnate) [reply rn/Teacher/package.json:35]: oh yeah, it's just not released yet. I couldn't figure out how to tell yarn to use "master".

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/51034583/p/fee793111cf55ed31f348fe5ef107235">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/51034587/p/4014736766c49658383dfbdca53563fc">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/51034563/p/1990af973005d2fef2ba0f7a4d244054">
    
  </a>
</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 92.93% | 92.93% | 0%

[MBL-14777](https://instructure.atlassian.net/browse/MBL-14777)

#### Affected Apps: None
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: rn/Teacher/package.json (hunk 3 above)
Comment (vanwagonet) [reply rn/Teacher/package.json:35]: if you put in the repo name instead of the version number, it will use master, just like it did for the fork ;)

**Comment by armstrongnate** [comment]
We're going to wait on this and give the others time to patch the fix.

**Pull request closed without merging**